packages/api-explorer/src/components/SchemaTabs/index.jsx

```diff
@@ -91,23 +91,17 @@ export default class SchemaTabs extends Component {
 
   renderSchemaExample() {
     const {schema} = this.state
-    let example = get(schema, EXAMPLE)
-
-    if (schema.type === 'array') {
-      example = get(schema.items, EXAMPLE)
-    }
-
     let generatedSchema
     try {
-      generatedSchema = generateFakeSchema(example)
```

Comment:
Was the function already aware of the fact that a schema can have its own examples?

We're still showing existing example when present, right?

Reply:
now each `examples` is shown on preview area, if present on schema, otherwise it is generated automatically from jsf and injected in the preview